stable/prometheus-operator/values.yaml

```diff
@@ -1537,7 +1540,7 @@ prometheus:
     ##
     image:
       repository: quay.io/prometheus/prometheus
-      tag: v2.17.2
+      tag: v2.18.0
```

Comment:
We should use 2.18.1 for the included bugfixes

Reply:
Done